user_manual/android_app.rst

```diff
@@ -95,23 +95,23 @@ Android apps, such as the Gallery app.
 
 .. figure:: images/android-4.png 
    :alt: Your ownCloud Files page.
-
-The ownCloud menu at the top left shows the name of the logged-in user, has a 
-shortcut to your files, the **Uploads** button, and the **Settings** button.
-
-The Uploads page contains a list of your recently-uploaded files, and a Retry 
-option for any failed uploads.
-
-.. figure:: images/android-5.png
-   :alt: Top-left menu.
-
+   
 Click the overflow button at the top right (that's the one with three vertical 
 dots) to open a user menu. **Grid view** toggles between grid and list 
 view. **Refresh account** syncs with the server, and **Sort** 
 gives you the option to sort your files by date, or alphabetically.
 
 .. figure:: images/android-6.png
-   :alt: Top-right menu.
+   :alt: Top-right menu.   
+
+The ownCloud menu at the top left shows the name of the logged-in user, has a 
+shortcut to your files, the **Uploads** button, and the **Settings** button.
+
+The Uploads page displays the status of files currently uploading, a list of 
+your recently-uploaded files, and a Retry option for any failed uploads.
```

Comment:
I can confirm, Carla. An upload to a folder where the user cannot write ends with a failed upload with "Permission error". If the users taps on it in the uploads view, the upload is directly retried, finishing again in "Permission error". Unless something is done outside the app to grant the permission, the upload will always fail